src/Sylius/Bundle/CoreBundle/Resources/config/services.xml

```diff
@@ -150,6 +150,12 @@
             <argument type="service" id="sylius.variant_resolver.default" />
         </service>
 
+        <service id="sylius.order_processing.order_exchange_rate_and_currency_updater" class="Sylius\Component\Core\OrderProcessing\OrderExchangeRateAndCurrencyUpdater">
```

Comment:
I'd be for removing all class parameters in favour of FQCN, but it's definitely not the the purpose or even related to this PR, so reverting for the time being.